src/renderer/services/midgard/pools.ts

```diff
@@ -167,16 +196,51 @@ const createPoolsService = (
     RxOp.shareReplay(1)
   )
 
-  // `TriggerStream` to reload detailed data of pool
-  const { stream$: reloadPoolDetail$, trigger: reloadPoolDetail } = triggerStream()
+  /**
+   * Loading queue to get all needed data for `PendingPoolsState`
+   */
+  const loadPendingPoolsStateData$ = (): PendingPoolsStateLD => {
+    const poolAssets$: PoolStringAssetsLD = FP.pipe(apiGetPoolsPending$, RxOp.shareReplay(1))
+    const assetDetails$ = getAssetDetails$(poolAssets$)
+    const poolDetails$ = getPoolDetails$(poolAssets$)
+
+    return FP.pipe(
+      combineLatest([poolAssets$, assetDetails$, poolDetails$]),
+      RxOp.map((state) => RD.combine(...state)),
+      RxOp.map(
+        RD.map(([poolAssets, assetDetails, poolDetails]) => {
+          return {
+            poolAssets,
+            assetDetails,
+            poolDetails
+          }
+        })
+      ),
```

Comment:
pls app to the `helpers/rx/liveData.ts:40` the following
```ts
sequenceS: sequenceS(instanceLiveData),
```
and after that we can use something like this

```ts
liveData.sequenceS({
        poolAssets: poolAssets$,
        assetDetails: assetDetails$,
        poolDetails: poolDetails$
      }),
```

Reply:
Great suggestion!